Gems/DiffuseProbeGrid/Code/Source/Components/DiffuseProbeGridSystemComponent.cpp

```diff
@@ -59,6 +59,7 @@ namespace AZ
 
         void DiffuseProbeGridSystemComponent::GetRequiredServices([[maybe_unused]] ComponentDescriptor::DependencyArrayType& required)
```

Comment:
nit: you can remove the `[[maybe_unused]]` now if you want

Reply:
Done